sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -1729,7 +1729,7 @@ public async Task CreateSnapshotUsingManualPolling()
                         ValidateCompletedOperation(operation);
                         break;
                     }
-                    await Task.Delay(500);
+                    await Task.Delay(125);
```

Comment:
Tweaked timing here as the test was consistently exceeding the local timeout.

Reply:
This can use RecordedTestBase.Delay to avoid delaying when in playback mode.

Reply:
Thanks!   I didn't know that was a thing.   Updated.